ocfweb/error404/er404.py

```diff
@@ -0,0 +1,9 @@
+from django.shortcuts import render
+
+
+# Function for rendering the error 404 page
```

Comment:
consider using a docstring instead of a comment; if you use a docstring, it gets jammed under `__doc__` for the function, which enables tools like `pydoc` and sphinx to generate useful documentation

these are standardized in https://www.python.org/dev/peps/pep-0257/